SparkPost/2web2ui
# [AC-1867] Global tracking for Support, Domains and Ippools forms

[AC-1867] Global tracking for Support, Domains and Ippools forms 

### What Changed

- Allowlisted forms for Support, Domains and Ipools
- Added cypress tests for Domains forms

### How To Test

- Check if all information from domains form is sent correctly to segment
- Check if all information from Support form is sent correctly to segment
- Check if all information from Ippools form is sent correctly to segment

Note - 
I haven't been able to verify two forms - namely premium-support-form and ippools-ip-form, facing issues pointing local to prod. (will try this again)


### To Do
